docs/components/wells.md

```diff
@@ -28,59 +28,8 @@ variations:
       </div>
     variation_description: ''
     variation_name: Default well
-usage: >-
-  #### Use cases
-
-
-  Use sparingly to attract users’ attention to a concise piece of content.
-  Ideally, content within a well should not take up more than a third of page
-  content.
-
-
-  Full width text, images, and embedded media (like video) may be placed within
-  wells, however, if content includes imagery also consider the featured content
-  module.
-
-
-  #### Content guidelines
-
-
-  * Grey wells should be used to feature content or specific calls to action
-
-  * Grey well will help visually set apart the content within it, so use it to
-  draw attention to that content.
-
-  * Grey wells should take up less than a third of a page.
-
-  * Currently, single column content only.
-
-
-  #### Behavior
-
-
-  ##### Breakpoints 900+
-
-
-  Page with a right sidebar
-
-
-  ![](/design-system/images/uploads/wells_behavior_mockup_1.png)
-
-
-  Browse page with left nav
-
-
-  ![](/design-system/images/uploads/wells_behavior_mockup_2.png)
-
-
-  ##### Breakpoints 900 and under
-
-
-  Wells are full width. At breakpoints 600 and under there are slight padding
-  adjustments to maximize real estate on smaller displays.
-
-
-  ![](/design-system/images/uploads/wells_behavior_mockup_3.png)
+usage: "#### Use cases\n\nUse sparingly to attract users’ attention to a concise piece of content. Ideally, content within a well should not take up more than a third of page content.\n\nFull width text, images, and embedded media (like video) may be placed within wells, however, if content includes imagery also consider the featured content module.\n\n#### Content guidelines\n\n* Grey wells should be used to feature content or specific calls to action; text should be succinct.\n* Grey well will help visually set apart the content within it, so use it to draw attention to that content.\n* Grey wells should take up less than a third of a page.\n* Currently, single column content only.\n* Types of atoms you can place in a well include:\n  * Headings (H2 - H5)\n  * Text (including bullets and numbered lists)\r\n  * Links\n* Headings should be as succinct as possible and 35 characters or less (including spaces).\n\n#### Behavior\n\n##### Breakpoints 900+\n\nPage with a right sidebar\n\n![](/design-system/images/uploads/wells_behavior_mockup_1.png)\n\nBrowse page with left nav\n\n![](/design-system/images/uploads/wells_behavior_mockup_2.png)\n\n##### Breakpoints 900 and under\n\nWells are full width. At breakpoints 600 and under there are slight padding adjustments to maximize real estate on smaller displays.\n\n![](/design-system/images/uploads/wells_behavior_mockup_3.png)"
```

Comment:
This renders okay, but let's format it for readability here by putting each section (####, etc.) on its own line, as it was prior to the edit.